src/components/sequencing/SequenceEditor.svelte

```diff
@@ -216,6 +218,15 @@
     }
   }
 
+  $: showOutputs = !inVmlMode() && !!outputFormats.length;
+  $: {
+    if (showOutputs) {
+      editorHeights = toggleSeqJsonPreview ? '1fr 3px 1fr' : '1.88fr 3px 80px';
+    } else {
+      editorHeights = toggleSeqJsonPreview ? '1fr 3px' : '1.88fr 3px';
```

Comment:
Do you need to handle this case if there's nothing to toggle if `showOutputs` is false?

Reply:
addressed in https://github.com/NASA-AMMOS/aerie-ui/pull/1472/commits/1e966d2de7457672427eb7b549b54fc20dc85284